cg/cli/upload/genotype.py

```diff
@@ -19,13 +22,12 @@ def genotypes(context, re_upload, family_id):
         _suggest_cases_to_upload(context)
         context.abort()
 
-    tb_api = context.obj["tb_api"]
     gt_api = context.obj["genotype_api"]
     hk_api = context.obj["housekeeper_api"]
     status_api = context.obj["status"]
     family_obj = status_api.family(family_id)
+    upload_genotypes_api = UploadGenotypesAPI(hk_api=hk_api, gt_api=gt_api)
+    results = upload_genotypes_api.data(family_obj.analyses[0])
 
-    api = UploadGenotypesAPI(status_api, hk_api, tb_api, gt_api)
-    results = api.data(family_obj.analyses[0])
     if results:
-        api.upload(results, replace=re_upload)
+        upload_genotypes_api.upload(results, replace=re_upload)
```

Comment:
When this is called, we are no longer going to see the logging from the GenotypesAPI, and when calledprocesserror will be raised upon failure,  there won't be additional feedback from genotypes like we have now. Perhaps could also try to pipe the logs to cg cli?

Reply:
This is being done in the Process module